engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
@kcibul A backend already knows that (the attempt # is embedded in the jobkey). And I would set this number to a number higher or equal to the largest that you've seen so far in your workloads. Just pick something and let me know, I'll update it's value here.